.github/workflows/vulnerability_scanning.yml

```diff
@@ -67,38 +55,17 @@ jobs:
           path: ./trivyignore
           ref: trivy-whitelist
 
-      - name: download docker image from minio
-        if: env.TRIVY_IMG_MODE
-        run: |
-          export AWS_ACCESS_KEY_ID=${{ secrets.MINIO_ACCESS_KEY_ID }}
-          export AWS_SECRET_ACCESS_KEY=${{ secrets.MINIO_SECRET_ACCESS_KEY }}
-          aws --endpoint-url ${{ secrets.MINIO_ENDPOINT_URL }} s3 cp s3://docker-img-artifacts/${{ inputs.image_tar_name }} ./
-
       - name: Run Trivy vulnerability scanner in image mode
-        if: env.TRIVY_IMG_MODE
         uses: aquasecurity/trivy-action@master
         with:
+          image-ref: ${{ secrets.REGISTRY }}/${{ inputs.project }}/${{ inputs.component }}:${{ inputs.docker_image_version }}
```

Comment:
mhh. how does it work? will it pull the image from harbor automatically ? how I didn't see you pass the username/password to it?

Reply:
coz of limit that can't pull image when harbor is in pending for scanning.choose to scan image locally.